storage/lastestData/latestDataProvider.go

```diff
@@ -1,4 +1,4 @@
-package factory
+package lastestData
```

Comment:
latestData instead lastestData. Also rename the folder

Reply:
renamed